src/cairoWriter/writers/returnWriter.ts

```diff
@@ -6,24 +6,17 @@ import { CairoASTNodeWriter } from '../base';
 import { getDocumentation } from '../utils';
 
 const KECCAK = 'keccak_ptr';
-const WARP_MEMORY = 'warp_memory';
 
 export class ReturnWriter extends CairoASTNodeWriter {
   writeInner(node: Return, writer: ASTWriter): SrcDesc {
     const documentation = getDocumentation(node.documentation, writer);
     const returns = node.vExpression ? writer.write(node.vExpression) : '()';
 
-    const finalizeWarpMemory = this.usesImplicit(WARP_MEMORY, node)
-      ? 'default_dict_finalize(warp_memory_start, warp_memory, 0);\n'
-      : '';
-
     const finalizeKeccakPtr = this.usesImplicit(KECCAK, node)
       ? 'finalize_keccak(keccak_ptr_start, keccak_ptr);\n'
       : '';
```

Comment:
Is this still necessary? I thought the initialization of these got removed from `getBody` in `cairoFunctionDefinitionWriter`

Reply:
Imo implicits deserve a separate PR so I only did the necessary changes to them in order to implement static arrays